src/helpers/inSaleTvs1e.js

```diff
@@ -0,0 +1,13 @@
+import {inventory} from "../constants/inventory.js";
+import countBoughtTvs1c from "./countBoughtTvs1c.js";
+import countSoldTvs1a from "./countSoldTvs1a.js";
+
+function toSellTvs() {
+    for (let i = 0; i < inventory.length; i++) {
+      return countBoughtTvs1c() - countSoldTvs1a()
+    }
```

Comment:
De for loop is overbodig, omdat je niets met `i` doet en het altijd direct returned
```suggestion
    return countBoughtTvs1c() - countSoldTvs1a();
```